scripts/upload-table-config.ts

```diff
@@ -19,8 +19,8 @@ import {
 // ---------------------------------------------------------
 
 const newTables: Table[] = [
-  ...generateTables({ num: 5, capacity: 6 }),
-  ...generateTables({ num: 5, capacity: 2, startFrom: 5, columns: 2 }),
+  ...generateTables({ num: 5 }),
+  ...generateTables({ num: 5, startFrom: 5, columns: 2 }),
```

Comment:
Was this change intentional?

Reply:
Yes, the capacity is now calculated based on columns\rows number 
![image](https://user-images.githubusercontent.com/63194656/145219238-1abedb7a-f705-4bfc-b1d5-fb1ee10d25f7.png)